### ISSUE TITLE
Page title based on heading title

### ISSUE THREAD

**Issue by jamesread** [owenh000/asciidoctor-multipage#45]
Hey, I've been using asciidoctor-multipage for several years now, for several projects, the biggest of which is  http://docs.olivetin.app ( https://github.com/OliveTin/docs.olivetin.app ), and as a developer myself I know that I appreciate when people say thank you, so, thank you! ;-) asciidoctor-multipage has made my life writing longer documentation much easier than having to use something like asciibinder. 

I'm not sure if this is a support request (because I've not found the configuration option yet), or a feature-request (if this isn't yet possible), but I'd like the page title to be reflected in the html - currently every page uses the document title, ie; 

```yaml
= OliveTin
:title: OliveTin documentation
:multipage-level: 2

[#installation]
== Installation guide
blah
```

Which renders: `<title>OliveTin documentation</title>` on installation.html. I would much rather something like; `<title>Installation &bull; OliveTin Documentation</title>` but just `<title>Installation</title>` would be good enough.

The reason being is that it helps people with lots of docs tabs open (often that is me), and helps differentiate search engine results. eg,  this isn't great;

![image](https://github.com/owenh000/asciidoctor-multipage/assets/2025715/a3dc1043-b634-4832-87e6-5ad1349695a1)

**Comment by owenh000** [comment]
Hi @jamesread, it looks like this is a feature request, so I've labeled it as such.

Issue #46 needs to be done first.

**Comment by jamesread** [comment]
@owenh000 Many thanks for the quick reply, and totally understand the need for #46 - I had a look through the code myself, but woah it's difficult to understand (purely due to my very poor understanding of Ruby, and zero understanding of Asciidoctor's internals. 

What sort of timeframe do you realistically think it will take for you to implement this?

**Comment by owenh000** [comment]
>What sort of timeframe do you realistically think it will take for you to implement this?

Without sponsors or some other arrangement, this is just a hobby project and I don't expect to have time to work on it at all aside from the issue discussions and any really simple bug fixes. Unfortunately some of the "that should be easy to fix" bugs are instead, after a closer look, pointing to the need for some overall clean-up!

If you are interested in sponsoring this project then please see [my Sponsors page](https://github.com/sponsors/owenh000). And as noted there, you are welcome to contact me by email too.

**Comment by jamesread** [comment]
You have an email from me in your inbox, to ask about a one-time sponsorship for this issue :-)

**Comment by jamesread** [comment]
Hey, thanks for the discussion on email 2 years ago. I migrated to Antora a little while ago, which actually solved lots of different problems as well for me, so I no longer need this issue open. 

Thanks again for the discussion, and I hope that your project continues to go well - thanks very much! :-)

**Comment by owenh000** [comment]
@jamesread, thanks for the update and for closing the issue since the feature is no longer needed. Also thanks for mentioning Antora; I did not know about it. I will mention it in the README as an alternative.
